test/module/irohad/consensus/yac/network_test.cpp

```diff
@@ -43,13 +42,15 @@ namespace iroha {
 
           grpc::ServerBuilder builder;
           int port = 0;
-          builder.AddListeningPort(peer.address,
+          builder.AddListeningPort("0.0.0.0:0",
```

Comment:
This change is not part of this PR, I don't know how these commits ended up here, I guess I need to rebase :)

Also I think removing hardcoded literals like this one should be a separate issue, so that they all get organized in one place and used by all tests.

Reply:
The problem that you've introduced magic values here. It's better write clean code so there's no need in additional issues for fixing tech debt